playbooks/common/openshift-master/config.yml

```diff
@@ -304,6 +231,30 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
+# Create master certificates separately to speed up configuring
+# masters (since they are configured serially). The
+# openshift_master_certificates role may be added as a dependency of
+# the openshift_master role once masters are configured in parallel.
```

Comment:
If there are problems starting master services in parallel then my serial command hack won't work in this case.